commands/audit/audit.go

```diff
@@ -79,6 +86,16 @@ func (auditCmd *AuditCommand) CreateXrayGraphScanParams() *services.XrayGraphSca
 	}
 	params.IncludeVulnerabilities = auditCmd.IncludeVulnerabilities
 	params.IncludeLicenses = auditCmd.IncludeLicenses
+	params.MultiScanId = auditCmd.analyticsMetricsService.GetMsi()
+	xscManager := auditCmd.analyticsMetricsService.XscManager()
+	if xscManager != nil {
+		version, err := xscManager.GetVersion()
+		if err != nil {
+			log.Debug(fmt.Sprintf("Can't get XSC version for xray graph scan params. Cause: %s", err.Error()))
+		}
+		params.XscVersion = version
+	}
```

Comment:
```suggestion
	if xscManager := auditCmd.analyticsMetricsService.XscManager(); xscManager != nil {
		version, err := xscManager.GetVersion()
		if err != nil {
			log.Debug(fmt.Sprintf("Can't get XSC version for xray graph scan params. Cause: %s", err.Error()))
		}
		params.XscVersion = version
	}
```

Reply:
The way I implemented it is totally fine as well.